docs/modules/ROOT/pages/how-tos/cloudscale/decommission.adoc

```diff
@@ -1,5 +1,6 @@
 = Uninstallation on cloudscale
 
+:provider: cloudscale
 :lb_tf_resource: module.cluster.module.lb.cloudscale_server.lb
```

Comment:
Remove these and add them in the partial since we disambiguate by `provider` now.

```suggestion
```